schema/release-schema.json

```
@@ -1211,11 +1211,11 @@
     "Budget": {
       "type": "object",
       "title": "Budget information",
-      "description": "This section contains basic information about the budget estimated for, or allocated to, this contracting process at the present time. Further documentation and data about how budgets have been allocated to a contracting process should be published outside of OCDS data, according to the best available standards.",
+      "description": "This section contains basic information about the budget that will be used in a future contracting process. Further documentation and data about how budgets have been allocated should be published outside of OCDS data, according to the best available standards.",
       "properties": {
         "id": {
           "title": "ID",
-          "description": "An identifier for the budget line item which provides funds for this contracting process. This identifier should be possible to cross-reference against formal budget documents.",
+          "description": "An identifier for the budget line item which will be used in a future contracting process. This identifier should be possible to cross-reference against formal budget documents.",
```

Good god, we really didn't use consistent wording (budget, budget line item, budget allocation).

```suggestion
          "description": "An identifier of the budget to be used in a future contracting process. This identifier should be possible to cross-reference against formal budget documents.",
```